.github/workflows/tf-aws-test.yml

```
@@ -39,21 +42,45 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
+
       - name: Setup Node.js v18
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: ${{ env.NODE_VERSION }}
+
+      ##################### install a local wing version #########################
+      - name: Download Dist Artifacts
+        if: ${{ env.LOCAL_BUILD ==  'true' }}
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist
+
+      - name: Install Wing from Dist
+        if: ${{ env.LOCAL_BUILD ==  'true' }}
+        run: |
+          mkdir temp
+          cd temp
```

Maybe make it more descriptive

```suggestion
          mkdir localwing
          cd localwing
```